File: arrays 101/merge_sorted_arrays.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
                
        for i in range(len(nums1)-m):
            del nums1[len(nums1)-1]
            
        for i in range(len(nums2)-n):
            del nums2[len(nums2)-1]
                
        for i in nums2:
            nums1.append(i)
        
        nums1.sort()

File: arrays 101/merge_sorted_arrays .java
import java.util.*;
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        
        int i=0, j=0, len=0;
        int[] nums3 = new int[nums1.length];
        
        while(i<m && j<n)
        {
            if (nums1[i]<nums2[j])
            {
                nums3[len++] = nums1[i++];
            }
            else
            {
                nums3[len++] = nums2[j++];
            }
        }
        
        
        while(i<m)
        {
            nums3[len++] = nums1[i++];
        }
        
        while(j<n)
        {
            nums3[len++] = nums2[j++];
        }
        
        for(int k=0; k<nums1.length; k++)
        {
            nums1[k] = nums3[k];
        }
    }
}